src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import { Calculator } from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="App-Calculator">
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] Kindly make the `App` component a class-based component as it is a project requirement. You could do this:
```
class App extends Component {
  render() {
    // reference as a js object
    return {
  <div className="App-Calculator">
      <Calculator />
  </div>
    };
  }
}
```